### DISCUSSION TITLE
Talkgroup Priority Suggestion

### DISCUSSION THREAD

**Discussion by steve-horvath** [chuot/rdio-scanner#234] [Ideas]
How about adding a Priority level to talkgroups?

During a busy time on the radio, I may get the queue backed up for dozens, if not over a hundred calls (with time delays of 10-20 minutes), and I need to sit listening to the calls, and hitting next to skip over non-important calls or talkgroups I am less concerned with hearing.

The priority value would allow me to automatically skip lower priority items during the live feed.

For example, if I set the priority to Fire Dispatch to 5, and the priority of the local parks department to 20, and there are 15 calls in the queue, the parks department would be skipped, but if there were no calls in the queue, I would hear all the traffic.

**Comment by chuot** [comment]
Priority is planned for future release where it will be configurable on the client side only (not on server). PRI on a TG will have the call queued for playing as soon as possible, after other PRI calls have been played.

Until this feature is being implemented, try using the new temporary avoid feature on busy nights.
